src/NuGet.Clients/NuGet.PackageManagement.UI/Models/DetailControlModel.cs

```diff
@@ -141,7 +141,7 @@ public virtual void CleanUp()
             }
 
             // Add Current package version to package versions list.
-            _allPackageVersions = new List<NuGetVersion>() { searchResultPackage.Version };
+            _allPackageVersions = new List<NuGetVersion>() { searchResultPackage.LatestVersion };
```

Comment:
As discussed offline, please take appropriate call here to continue with this code or have minimum change for now but log another workitem to clean it later.